### PR TITLE
DEVDOCS-2531-remove-coupon-array-under-line-items

### PR DESCRIPTION
**Pull request by traci707**
[DEVDOCS-2531](https://jira.bigcommerce.com/browse/DEVDOCS-2531)
Removed coupon array under cart physical and digital line items to correct the Response schema

Two changes:
1. Removed lines 1058 to 1079 (array) and replaced with line 1030 (object).
2. Removed lines 1212 to 1233 (array) and replaced with line 1153 (object).

### CODE REVIEW

**Review by capsula4 — APPROVED** [review]
👏

File: reference/checkouts.v3.yml [hunk 1]
```diff
@@ -1048,35 +1022,12 @@ definitions:
...
                             discounted_amount:
-                              description: >-
-                                The discounted amount applied within a given
-                                context.
+                              description: The discounted amount applied within a given context.
                               type: number
                               format: double
                       coupons:
                         description: ''
-                        type: array
```
Comment (ektravel) [inline reference/checkouts.v3.yml:1058]: I believe this should be `type: object`. The AppliedCoupon model has all of the applicable properties: id, code, discounted_amount, coupon_type.

File: reference/checkouts.v3.yml [hunk 2]
```diff
@@ -1202,35 +1145,12 @@ definitions:
...
-                              type: string
-                            discounted_amount:
-                              description: >-
-                                The discounted amount applied within a given
-                                context.
-                              type: number
-                              format: double
-                          required:
-                            - code
+                        type: number
```
Comment (ektravel) [inline reference/checkouts.v3.yml:1153]: I believe this should be `type: object`. The AppliedCoupon model has all of the applicable properties: id, code, discounted_amount, coupon_type.

**Review by ektravel — COMMENTED** [review]
Added my comments.

**Comment by traci707** [comment]
Reviewed server-to-server carts and storefront checkouts as well. I made the following changes:
Storefront:
In Checkout-Resp, 
- changed "TaxIncluded" to "isTaxIncluded"
- added "isMutable", but I do not know what to put for definition.
In Checkout-Full,
- added "comparisonPrice"
- added "extendedComparisonPrice"

Server-to-server:
Physical, digital, gift certificates, and custom_items are good now. The array has been removed.

File: reference/checkouts.sf.yml [hunk 3]
```diff
@@ -3609,6 +3612,9 @@ definitions:
                             description: Quantity of this item.
                             type: number
                             format: double
+                          brand:
+                            description:
+                            type:
```
Comment (ektravel) [inline reference/checkouts.sf.yml:3617]:
```suggestion
                            type: string
```

File: reference/checkouts.sf.yml [hunk 4]
```diff
@@ -3609,6 +3612,9 @@ definitions:
                             description: Quantity of this item.
                             type: number
                             format: double
+                          brand:
+                            description:
```
Comment (ektravel) [inline reference/checkouts.sf.yml:3616]:
```suggestion
                            description: The product brand.
```

**Review by ektravel — COMMENTED** [review]
Added description and type for `brand` property.

**Review by markcmurphy — APPROVED** [review]

**Comment by aglensmith** [comment]
Closing due to conflicts. Reopen or push once conflicts are resolved.

**Pull request merged**